php/class-customize-snapshot-manager.php

```diff
@@ -1506,6 +1508,41 @@ public function render_templates() {
 				<p>{{ data.message }}</p>
 			</div>
 		</script>
+
+		<script type="text/html" id="tmpl-snapshot-conflict-button">
+			<?php $title_text = __( 'click to expand', 'customize-snapshots' ); ?>
+			<# id= data.setting_id.replace( /]/g, '' ).split( '[' ).filter( Boolean ).join( '-' ); #>
+			<span>
+				<?php esc_html_e( 'Snapshot conflicts', 'customize-snapshots' ); ?>
+				<a href="#TB_inline?width=600&height=550&inlineId=snapshot-conflicts-{{id}}" class="dashicons dashicons-warning thickbox snapshot-conflicts-button" title="<?php echo esc_attr( $title_text ); ?>"></a>
```

Comment:
This string: `#TB_inline?width=600&height=550&inlineId=snapshot-conflicts-{{id}}` should be entity-escaped (e.g. `&amp;`)